contracts/collection.scilla

```diff
@@ -338,5 +372,24 @@ transition TokenSaleCallback(
     end
 end
 
+(* Sets `max_commission_fee_bps` *)
+(* @Requirements: *)
+(* - `_sender` must be the contract owner. Otherwise, it must throw `NotContractOwner` *)
+(* - `new_max_commission_fee_bps` must be in the range of 0 and 2500. Otherwise, it must throw `InvalidMaxFeeBPS` *)
+transition SetMaxCommissionFeeBPS(new_max_commission_fee_bps: Uint128)
+  RequireContractOwner;
+  RequireValidMaxBPS new_max_commission_fee_bps;
+
+  (* Can we run into an issue where the combined fees (royalty, service & commission) result in gt 100%? *)
```

Comment:
@sam-zilliqa this might be possible as currently the max royalty fee is not checked in Fixed Price (in English auction, I think it's max 25%). But IMO this logic should be handled in  Fixed Price/Auction contracts

Reply:
But setting to max 25% is good, also consistent with max royalty.